docker/test/Dockerfile.ubuntu18.04

```diff
@@ -29,15 +29,17 @@ RUN apt-get update && apt-get -y install \
       libcppunit-dev \
       patch \
       python3-dev \
+      python3-venv \
       wget \
       zip \
-      virtualenv \
       unzip \
       git \
       curl \
       tree \
       openjdk-11-jdk-headless
 
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 10
```

Comment:
is linking `python` to `python3` needed - was it that transient dependency which needed it?

Reply:
I got this error:
```
ERROR: An error occurred during the fetch of repository 'pip_deps':
   pip_import failed:  (src/main/tools/process-wrapper-legacy.cc:75: "execvp(python, ...)": No such file or directory
)
ERROR: no such package '@pip_deps//': pip_import failed:  (src/main/tools/process-wrapper-legacy.cc:75: "execvp(python, ...)": No such file or directory
)
```

Reply:
That makes it sound like bazel (that `.cc` is a part of it) is trying to run `python` on the system, rather than `python3` for some reason, which would be a preexisting issue. Maybe it's because `test --host_force_python=PY3` isn't in `tools/bazel.rc`?

Reply:
Just tested with the following, but it still fails:
```
test --genrule_strategy=standalone
test --host_force_python=PY3
test --ignore_unsupported_sandboxing
test --spawn_strategy=standalone
```

Reply:
Seems to be related to this line. Maybe we don't need it? Testing it now with it removed. https://github.com/apache/incubator-heron/blob/ee73293b2ce5f1c0aa0506a4398bb2efe696944f/WORKSPACE#L398

Reply:
Looks like it's needed for some of the `tar` and `docker` rules.